### ISSUE TITLE
mpl and colorcet names

### ISSUE THREAD

**Issue by vascotenner** [bokeh/colorcet#2]
I noticed that both mpl and colorcet uses the name 'inferno' for two different colormaps. The cc inferno map looks very similar to the mpl plasma map.

https://bokeh.github.io/colorcet/
http://matplotlib.org/users/colormaps.html

**Comment by ArchangeGabriel** [comment]
It’s even more confusing since both `inferno` are listed under `https://bokeh.github.io/colorcet/#Testing-perceptual-uniformity`. BTW, I should note here that I don’t agree with this sentence “Here the inferno map seems to have better discriminability than the viridis map, despite both being perceptually uniform.”. Maybe an issue with screen calibration? Or just a difference between peoples.

But I’m not sure how much an issue this is, and how it might be fixed: changing the name means breaking compatibility.

**Comment by bryevdv** [comment]
ping @jbednar

**Comment by jbednar** [comment]
Strange, I don't seem to have gotten a notification for there being an issue on this repo!  They don't come up often, so I haven't been checking explicitly.

When used with mpl, the colorcet maps are all prefixed with "cet_" (i.e., "cet_inferno"), to avoid any ambiguity.  But I'm happy to change the name "inferno" to the more literal "bmy" to avoid confusion.  It's true that it breaks compatibility, but this is precisely the sort of issue that made me call the version 0.9 rather than 1.0 -- I wanted a chance to fix things. Anyone have any other thing they want to change before officially declaring it 1.0?  E.g. the "blues" colormap could be confused with "Blues" from matplotlib?

As for the statement "the inferno map seems to have better discriminability than the viridis map, despite both being perceptually uniform", I just looked at that section on three different monitors, and asked a friendly nearby graphic designer to do the same, and in all three cases for both observers we found the "comb teeth" extending further down for "inferno" compared to "viridis".  So  yes, screen calibration or observer could be an issue, but the claim is true at least for these three very different looking monitors and two observers...
